assets/javascripts/discourse/services/d-templates-modal.js

```diff
@@ -0,0 +1,24 @@
+import { action } from "@ember/object";
+import Service from "@ember/service";
+import { tracked } from "@glimmer/tracking";
+
+export default class DTemplatesModalService extends Service {
+  @tracked showing = false;
+  @tracked model = null;
+
+  @action
+  show(model) {
+    this.showing = true;
+    this.model = model;
```

Comment:
We could drop `showing`, and instead do

```
{{#if model}}
```
in the template?

Reply:
Nice catch. Done!